app/components/header.hbs

```diff
@@ -51,6 +51,12 @@
           class="nav__element"
           href={{this.STATUS_URL}}
         >Status</a></li>
+        {{!-- {{#if @dev}} --}}
```

Comment:
If not needed please remove this comments

Reply:
okay